tendrl-node-agent.service

```diff
@@ -3,11 +3,12 @@ Description= A python agent local to every managed storage node in the sds clust
 
 [Service]
 Type=simple
+Environment="HOME=default_foo_value"
```

Comment:
What's with the value for $HOME here? Also, how was it figured out that $HOME is required?